var/spack/repos/builtin/packages/dla-future/package.py

```diff
@@ -44,6 +44,9 @@ class DlaFuture(CMakePackage, CudaPackage, ROCmPackage):
     depends_on("mpi")
     depends_on("blaspp@2022.05.00:")
     depends_on("lapackpp@2022.05.00:")
+
+    depends_on("blas")
+    depends_on("lapack")
```

Comment:
@albestro is CPU blas unconditionally required? Or is it redundant when targeting the GPU?

Reply:
Actually this already depends on the wrapper `blaspp` / `lapackpp`, so I think this is redundant?

Reply:
I'm confused :laughing: it uses blaspp and lapackpp, but also directly uses the underlying blas / lapack library?

Reply:
I didn't do any deep research for these packages, but I just saw blas/lapack being mentioned several times in the recipe without being a dependency, so added them.

Reply:
`dla-future` uses blas through `blaspp` (equivalently for `lapack`/`lapackpp`), but it also depends directly on the underlying implementation because we access some specific call of the implementation itself.

At the moment I don't recall exactly why and if it is needed to have a "direct" dependency. Probably @msimberg or @rasolca  remembers more?